docs/data-sources/aws_bucket_policy.md

```diff
@@ -5,6 +5,8 @@ subcategory: "Deployment"
 
 This datasource configures a simple access policy for AWS S3 buckets, so that Databricks can access data in it.
 
+-> This data source can be used with an account or workspace-level provider.
```

Comment:
```suggestion
-> This data source can be used with an account-level provider.
```
this data source only makes sense for account-level context

Reply:
It really doesn't require any provider. We already tried to make it account-only and got a pushback from users.